workers/unity/Assets/Gdk/Core/Tests/Editmode/Worker/WorkerRegistryTest.cs

```diff
@@ -7,17 +8,6 @@ namespace Improbable.Gdk.Core.EditmodeTests
     [TestFixture]
     public class WorkerRegistryTest
     {
-        internal class UnityTestWorker : WorkerBase
-        {
-            public const string WorkerType = "UnityTestWorker";
-
-            public override string GetWorkerType => WorkerType;
-
-            public UnityTestWorker(string workerId, Vector3 origin) : base(workerId, origin)
-            {
-            }
-        }
-
         [OneTimeTearDown]
```

Comment:
Do we have a defined ordering of Setup, Tests and Teardown? 

I would assume:
`OneTimeSetup`
`Setup`
`Tests`
`Teardown`
`OneTimeTeardown` ?

Reply:
Close :)

See https://docs.google.com/document/d/1fVOAYoEAW5cLxltKIiqT73A1_O8EMjgGP2BfOEHwj4I/edit#bookmark=id.7k40ko95rouf :

- one time set up and tear down
- per test set up and tear down
- test
- test

To be moved into a .md file in this repo

Reply:
That seems sensible to me :)